src/main/java/com/google/edith/ShelfDataReader.java

```diff
@@ -0,0 +1,97 @@
+package com.google.edith;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+/** Processes data file of product expiration information to populate future user grocery lists. */
+public class ShelfDataReader {
```

Comment:
Since this class doesn't have any instance variables, would it be a good idea to make all the methods public to avoid having to instantiate this class each time you need it?